app/javascript/src/components/TaxStatus.vue

```diff
@@ -17,43 +17,50 @@
     <div v-if="showTaxStatus">
       <div class="flex justify-between">
         <span class="w-1/3"></span>
-        <span class="w-1/3">Power points</span>
-        <span v-if="game.baseGame === 'imperial'" class="w-1/3">Tax chart position</span>
+        <span class="w-1/5">Flags</span>
+        <span class="w-1/5">Tax amount</span>
+        <span class="w-1/5">Power Points</span>
       </div>
       <div v-for="[nation,] of game.nations" :key="nation.value" class="flex justify-between">
         <span class="w-1/3">
           <strong>{{ stringify(nation.value) }}</strong>
         </span>
-        <span class="w-1/3">
-          {{ nextTaxationPowerPoints(nation) }}
-        </span>
-        <span v-if="game.baseGame === 'imperial'" class="w-1/3">
-          {{ nextTaxChartPosition(nation) }}
-        </span>
+        <span class="w-1/5"> {{ flagsPlaced(nation) }} </span>
+        <span class="w-1/5"> {{ nextTaxAmount(nation) }} </span>
+        <span class="w-1/5"> {{ nextTaxationPowerPoints(nation) }} </span>
       </div>
       <p class="text-sm">This shows the power points and tax chart positions of each nation if they were to tax <i>right now</i>.</p>
     </div>
   </div>
 </template>
 
 <script>
-import { nextTaxChartPosition, nextTaxationPowerPoints } from "../taxChartHelpers.js";
+import { nextTaxationPowerPoints } from "../taxChartHelpers.js";
```

Comment:
If we're removing `nextTaxChartPosition` from here I believe we can remove it from `taxChartHelpers.js` too, as I don't think it's used elsewhere.

Reply:
Also addressed in a future PR (665 I think)